cmd/access/node_builder/access_node_builder.go

```diff
@@ -711,13 +712,13 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 
 					checkpointHeight := builder.SealedRootBlock.Header.Height
 
-					bootstrap, err := pStorage.NewRegisterBootstrap(pdb, checkpointFile, checkpointHeight, builder.Logger)
+					buutstrap, err := pStorage.NewRegisterBootstrap(pdb, checkpointFile, checkpointHeight, builder.Logger)
```

Comment:
😆

Reply:
namespace overlap 😛

Reply:
https://media4.giphy.com/media/x2NZByIJT0WW0BPayC/giphy.gif

Reply:
:D maybe `regBootstrap`?

Reply:
leaving this for comic relief.